src/frontend/apps/desk/src/features/teams/components/PanelActions.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button } from '@openfun/cunningham-react';
+import React, { CSSProperties } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { Box, StyledLink } from '@/components';
+import { useCunninghamTheme } from '@/cunningham';
+
+import { TeamsOrdering } from '../api/useTeams';
+import IconAdd from '../assets/icon-add.svg';
+import IconSort from '../assets/icon-sort.svg';
+import { useTeamStore } from '../store/useTeamsStore';
+
+const ButtonSort = styled(Button)<{
+  $background: CSSProperties['background'];
+  $color: CSSProperties['color'];
+}>`
+  &.c__button {
+    svg {
+      background-color: transparent;
+      transition: all 0.3s;
+    }
+
+    &.c__button--active svg {
+      background-color: ${({ $background }) => $background};
+      border-radius: 10rem;
+      color: ${({ $color }) => $color};
+    }
+  }
+`;
+
+export const PanelActions = () => {
+  const { t } = useTranslation();
+  const { changeOrdering, ordering } = useTeamStore((state) => state);
```

Comment:
ditto, again I am a noob with zustand, is smht as `const { changeOrdering, ordering } = useTeamStore();`would work?

Reply:
https://github.com/numerique-gouv/people/pull/31/commits/eb7af0f41bfd043adf1a6ae180597e6a50e1ae6f